examples/OECC/scripts/stop.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+echo Stopping PASOE instance oepas1...
+sudo DLC=/psc/dlc /psc/wrk/oepas1/bin/tcman.sh stop
```

Comment:
Also, might be better to change `/psc/wrk/oepas1` to instead be a `$CATALINA_BASE` which gets set somewhere within the `rc.local` or other common script.

Reply:
rc.local cannot be used to set environment variables.
~/.bashrc could be used but it would be confusing as it is a hidden file.

Updated scripts to use $WRKDIR.